.gitignore

```diff
@@ -16,4 +16,5 @@ cmd/gf/gf
 temp/
 go.work
 go.work.sum
-!cmd/gf/go.work
\ No newline at end of file
+!cmd/gf/go.work
+/cmd/gf/internal/cmd/testdata/genctrl/api/article/article.go
```

Comment:
Why ignoring for this file?

Reply:
This file is `gf gen ctrl` to generate. It will compare with `/cmd/gf/internal/cmd/testdata/genctrl/api/article/article_expect.go.txt`

Reply:
Have improved.